package.json

```diff
@@ -70,9 +70,12 @@
     "@google-cloud/logging-winston": "^3.0.6",
     "@google-cloud/profiler": "^3.0.1",
     "@google-cloud/trace-agent": "^4.2.5",
+    "agentkeepalive": "^4.1.0",
     "express": "^4.17.1",
     "express-async-handler": "^1.1.4",
     "express-winston": "^4.0.3",
+    "superagent": "^5.2.2",
+    "superagent-cache-plugin": "^3.0.0-beta.1",
```

Comment:
FWIW, I had a lot of trouble using the beta of this plugin, I really this we should try to stick to the 2.x branch until the final release is ready. The caching dynamics are totally changed - it's now based on headers/the response from the server and setting a custom cache time has no effect.

Reply:
Oh, I never noticed it had installed the wrong one! Great catch, @jeresig